packages/venia-ui/lib/components/SearchPage/searchPage.js

```diff
@@ -75,7 +73,7 @@ const SearchPage = props => {
         );
     }
 
-    const totalCount = data ? data.products.total_count : 0;
+    const totalCount = data.products.total_count || 0;
```

Comment:
In a previous team discussion, I remember we wanted the ternary operator instead of the logical operators in such cases. I might be wrong, ignore if so.

Reply:
The ternary is unnecessary because `data` must exist for execution to reach here.